service/business/impl/getUserCompanies.go

```diff
@@ -28,7 +28,7 @@ func (a businessService) GetUserCompanies(w http.ResponseWriter, r *http.Request
 		return errors.NewServerError(model.GetUserCompaniesError, err.Error())
 	}
 
-	var companies []model.Company
+	var companies = make([]model.Company, 0)
```

Comment:
ini kenapa ditambahin make?

Reply:
Ini menjawab response API biar tetep list in case hasil dari query masih kosong

Reply:
oke noted.